src/screens/token/components/delegate-token/components/tabs/account-tab/account-tab.tsx

```diff
@@ -83,6 +86,7 @@ export class AccountTabComponent extends React.Component<
             .catch();
```

Comment:
Maybe send some Sentry errors if needed? In order not to have empty catches
If this was not already handled.